src/main/java/com/example/accommodationbookingapp/dto/accommodation/UpdateAccommodationRequestDto.java

```diff
@@ -0,0 +1,19 @@
+package com.example.accommodationbookingapp.dto.accommodation;
+
+import com.example.accommodationbookingapp.model.Accommodation;
+import jakarta.validation.constraints.Positive;
+import java.math.BigDecimal;
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class UpdateAccommodationRequestDto {
+    private Accommodation.Type type;
+    private String location;
+    private String size;
```

Comment:
i believe input data validation is required for update request as well, so it's better not to skip these fields